src/pages/FirstUsePage.js

```diff
@@ -83,7 +83,19 @@ export class FirstUsePageComponent extends React.Component {
     }
   }
 
-  onChangeText = text =>
+  onChangeServerUrl = text =>
+    this.setState({
+      serverURL: text,
+      status: 'uninitialised',
```

Comment:
:nit the string has been used three times, would be nice to have as a CONST

Reply:
I'm gonna quickly do this!